protocol/daemons/pricefeed/client/sub_task_runner.go

```diff
@@ -340,16 +340,22 @@ func RunMarketParamUpdaterTaskLoop(
 	}
 
 	// Update shared, in-memory config with the latest market params. Report update success/failure via logging/metrics.
-	err = configs.UpdateMarkets(getAllMarketsResponse.MarketParams)
-	if err == nil {
+	marketParamErrors, err := configs.UpdateMarkets(getAllMarketsResponse.MarketParams)
+	if err == nil && len(marketParamErrors) == 0 {
```

Comment:
Feel like it's misleading count "one market fails to update" as general "market update fails". Consider:
- gov introduces invalid oracle market "FOO-USD". It's never removed
- gov introduces another valid oracle market "OSMO-USD". It's actually successfully applied. 

Perhaps makes sense to add a `marketParam.Id` label to this counter?

Reply:
Good point! I updated the metrics code to emit 1 count for each market individually as success or error.